### PR TITLE
Added Mappls MapmyIndia

### PR DESCRIPTION
**Pull request by Saksham66**
- Added Support to launch Mappls App in Android and IOS

### CODE REVIEW

**Comment by mattermoran** [comment]
Hey @Saksham66 
Thanks for contributing!

I noticed your editors reformatted existing code probably because you have different lineLength. 
If you using vscode I just pushed a commit with vscode settings file so if you re-save all the files it should format it back. 

Otherwise the code will keep being reformatted back and forth :)

File: lib/src/directions_url.dart [hunk 1]
```diff
@@ -133,21 +133,21 @@ String getMapDirectionsUrl({
...
+    //     'dll': '${destination.latitude},${destination.longitude}',
+    //     'daddr': destinationTitle,
+    //     'sll': Utils.nullOrValue(
+    //       origin,
+    //       '${origin?.latitude},${origin?.longitude}',
+    //     ),
+    //     'saddr': originTitle,
+    //     'type': Utils.getMapsMeDirectionsMode(directionsMode),
+    //   },
+    // );
```
Comment (mattermoran) [inline lib/src/directions_url.dart:150]: I don't think this is standart dart formatting. probably some other plugin gets in the way?

Comment (Saksham66) [reply lib/src/directions_url.dart:150]: Hi @mattermoran 

I have used Android Studio plugin and I don't do any reformatting just add my changes and pushed that code

Comment (mattermoran) [reply lib/src/directions_url.dart:150]: I see. I guess then android studio does some auto formatting on save. If you can't fix it it's fine I'll merge and fix myself. 

I will take a look and merge it this weekend. In the meanwhile you can install and use directly from your fork.

**Comment by mattermoran** [comment]
this has been published as 3.5.0, cheers!

**Pull request closed without merging**